File: mini-era/cv/svm/src/matlab/randWrapper.m
function out = randWrapper(m,n)

out = zeros(m,n);

seed = 0.9;
for i=1:m
    for j=1:n
        if(i<j)
            out(i,j) = seed * (i/j);
        else
            out(i,j) = seed * (j/i);
        end
    end
end

end

File: mini-era/cv/common/randWrapper.c
/********************************
Author: Sravanthi Kota Venkata
********************************/

#include "sdvbs_common.h"

F2D* randWrapper(int m, int n)
{
    F2D *out;
    float seed;
    int i,j;

    out = fSetArray(m, n, 0);
    seed = 0.9;

    for(i=0; i<m; i++)
    {
        for(j=0; j<n; j++)
        {
            if(i<j)
                subsref(out,i,j) = seed * ((i+1.0)/(j+1.0));
            else
                subsref(out,i,j) = seed * ((j+1.0)/(i+1.0));
        }
    }

    return out;
}


